examples/flava/configs/finetuning/qnli.yaml

```diff
@@ -2,7 +2,7 @@
 _target_: definitions.FLAVAArguments
 training:
   _target_: definitions.TrainingArguments
-  lightning:
+  lightning_trainer:
```

Comment:
this renaming isnt needed? lets avoid for now

Reply:
My reason of renaming it from "lightning" - was to be explicit to clients that all the config arguments will go to the constructor arguments of LightningTrainer. As new config sections will have more details starting with lightning_ - for just. "lightning" can be confusing for clients if not to check the code. I am ok to leave it lightning for now :)

Reply:
yeah i understand the intention. just wary of making unrelated changes in the same PR